tf-build-tools/tf-test-groovy/src/main/groovy/com/deliveredtechnologies/terraform/helpers/Http.groovy

```diff
@@ -0,0 +1,37 @@
+package com.deliveredtechnologies.terraform.helpers
+
+import net.jodah.failsafe.Failsafe
+import net.jodah.failsafe.RetryPolicy
+import net.jodah.failsafe.function.CheckedSupplier
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import java.time.Duration
+import java.util.function.Consumer
+
+class Http {
+
```

Comment:
For helper classes, you should make the default constructor private, explicitly ensuring that an object cannot be created from the class (https://rules.sonarsource.com/java/tag/design/RSPEC-1118).

I was trying to think of a more elegant way to this without having everything static. And from a user interface perspective, nothing really seemed to make as much sense for what you are trying to do. Seems like maybe utility class may be the way to go.

Reply:
yea ill add a private constructor